keps/sig-api-machinery/3610-env-injection/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Env Injection
+kep-number: 3610
+authors:
+  - "@pacoxu"
+owning-sig: sig-api-machinery
```

Comment:
something related to injecting env into pods seems like it is owned by sig-node more than api-machinery (api-machinery could be participating since it might be involved in the mechanism)